src/components/constants.ts

```diff
@@ -148,64 +148,82 @@ export const TIME_UNIT_OPTIONS = [
 export const TEN_MINUTES_IN_MS = 1000 * 60 * 10;
 export const FIVE_MINUTES_IN_MS = 1000 * 60 * 5;
 
-const EXAMPLE_SCRIPT_SCRIPTED = btoa(`import { check, fail } from 'k6'
+const EXAMPLE_SCRIPT_SCRIPTED = btoa(`import { expect } from 'https://jslib.k6.io/k6-testing/0.5.0/index.js';
+import { check } from 'k6'
 import http from 'k6/http'
+import secrets from 'k6/secrets';
```

Comment:
Feels like this should be commented as it is not used in the script.

```suggestion
// import secrets from 'k6/secrets';
```

Reply:
Updated - thank you!